src/Appwrite/Platform/Workers/Migrations.php

```diff
@@ -34,6 +34,13 @@ class Migrations extends Action
 
     protected Document $project;
 
+    /**
+     * Accessed on cloud for telemetry.
```

Comment:
No need to mention cloud telemetry here, let's just say it's cached for performance reasons